### PR TITLE
FIX: add `seeg` to pick_types

### PR DESCRIPTION
**Pull request by kingjr**
Solves some bugs related to seeg data. Also see https://github.com/mne-tools/mne-python/issues/2548#issuecomment-149875453

### CODE REVIEW

**Comment by agramfort** [comment]
LGTM

however we should have a way to test this. So far our test suite does not handle seeg...

any idea of a stupid test maybe with rename_channels?

**Comment by dengemann** [comment]
> any idea of a stupid test maybe with rename_channels?

EpochsArray + arbitrarry data

**Comment by agramfort** [comment]
sounds like a plan

**Comment by kingjr** [comment]
shall I add separate tests with EpochsArray, or just add some seeg to an meeg data?

**Comment by agramfort** [comment]
just change as little code as possible to have this tested.

**Comment by kingjr** [comment]
is this error due to me?

File: mne/tests/test_epochs.py [hunk 1]
```diff
@@ -1790,4 +1792,15 @@ def test_add_channels():
...
 
 
+def test_seeg():
+    """Test the compatibility of the Epoch object with SEEG data."""
+    n_epochs, n_channels, n_times, sfreq = 5, 10, 20, 1000.
+    data = np.random.randn(n_epochs, n_channels, n_times)
+    events = np.array([np.arange(n_epochs), [0] * n_epochs, [1] * n_epochs]).T
+    info = create_info(n_channels, sfreq, 'seeg')
+    epochs = EpochsArray(data, info, events)
+    with warnings.catch_warnings(record=True):
```
Comment (Eric89GXL) [inline mne/tests/test_epochs.py]: please add a comment about why you needed to catch a warning

**Comment by Eric89GXL** [comment]
I don't think so. I restarted the builds. We'll have to keep an eye on other PRs, maybe Anaconda pushed an update that broke something.

File: mne/evoked.py [hunk 2]
```diff
@@ -841,9 +841,10 @@ def get_peak(self, ch_type=None, tmin=None, tmax=None, mode='abs',
             The time point of the maximum response, either latency in seconds
             or index.
         """
-        supported = ('mag', 'grad', 'eeg', 'misc', 'None')
+        supported = ('mag', 'grad', 'eeg', 'seeg', 'misc', 'None')
 
-        data_picks = pick_types(self.info, meg=True, eeg=True, ref_meg=False)
+        data_picks = pick_types(self.info, meg=True, eeg=True, seeg=True,
+                                ref_meg=False)
         types_used = set([channel_type(self.info, idx) for idx in data_picks])
```
Comment (jaeilepp) [inline mne/evoked.py]: You still need to add the ``seeg`` channel to lines 865-878. And while we are at it, perhaps the docstrings could be updated for the ``None`` parameters.

Comment (agramfort) [inline mne/evoked.py]: @jaeilepp don't hesitate to take over small PRs
if there is not much left.

**Pull request closed without merging**